src/Menu/Menu.js

```diff
@@ -125,10 +125,6 @@ class Menu extends Component<DefaultProps, AllProps, void> {
       // $FlowFixMe
       menuList.style.width = `calc(100% + ${size})`;
     }
-
-    if (this.props.onEnter) {
-      this.props.onEnter(element);
```

Comment:
No, I don't think that we should change this behavior.

> FYI: I am trying to make my own autocomplete field... and I would like to use Menu

The `Menu` component won't fit into a Autocomplete use case. I don't think that you should be using it.